src/test/java/uk/gov/hmcts/reform/blobrouter/reconciliation/service/SummaryReportServiceTest.java

```diff
@@ -124,7 +134,36 @@ void should_stop_process_if_parsing_supplier_statements_json_fails() throws Json
         verify(option).get();
         verifyNoMoreInteractions(option);
         verifyNoInteractions(envelopeService);
-        verifyNoInteractions(reconciliationReportRepository);
+        verify(reconciliationReportRepository).findByStatementId(envelopeSupplierStatement.id);
+        verifyNoMoreInteractions(reconciliationReportRepository);
+    }
+
+    @Test
+    void should_stop_process_if_parsing_supplier_statements_json_fails() throws JsonProcessingException {
```

Comment:
This scenario is not possible. The reconciliation endpoint will not save the invalid statement request

Reply:
on second thought, parsing the statement can still fail